plugins/importexport/crossref/classes/CrossRefExportDom.inc.php

```diff
@@ -328,14 +328,17 @@ function &_generateJournalArticleDom(&$doc, &$journal, &$issue, &$section, &$art
 		XMLCustomWriter::appendChild($journalArticleNode, $titlesNode);
 
 		/* AuthorList */
-		$contributorsNode =& XMLCustomWriter::createElement($doc, 'contributors');
-		$isFirst = true;
-		foreach ($article->getAuthors() as $author) {
-			$authorNode =& $this->_generateAuthorDom($doc, $author, $isFirst);
-			$isFirst = false;
-			XMLCustomWriter::appendChild($contributorsNode, $authorNode);
+		// Contributor list must not be empty
```

Comment:
No problem. This came from a conversation with Karl @ CrossRef - they do occasionally see empty <contributor/> nodes, which fails on their end. Technically this shouldn't be possible in OJS, as authors are mandatory (for now at least), but there are ways around it eg. via article import (I think?). There's probably a larger justification one could make for having articles without authors be possible, but that's another story.